<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XAI Viz</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Learning Rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Kernel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>600</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="blur">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Blur</string>
   </property>
  </widget>
  <widget class="QSlider" name="filter">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>600</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="choose_layer">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>45</y>
     <width>380</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Layer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kernel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>40</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Iterations</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="decay">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Decay</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generate_btn">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>620</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="freq_pen">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>370</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Frequency Penalization</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="learning_rate">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>40</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>65</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="iterations">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>40</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="selected_filter">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>630</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="vis_container">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>185</y>
     <width>350</width>
     <height>350</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>660</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Export Image</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="rotate">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>430</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_settings_btn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>470</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Export Settings</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="selected_neuron">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>690</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="neuron_select">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>660</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="neuron">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>660</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Neuron</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
